go.mod

```diff
@@ -14,8 +14,7 @@ require (
 	github.com/improbable-eng/grpc-web v0.15.0
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/onflow/cadence v0.42.5
-	github.com/onflow/flow-archive v1.3.4-0.20230503192214-9e81e82d4dcc
-	github.com/onflow/flow-core-contracts/lib/go/templates v1.2.4-0.20231016154253-a00dbf7c061f
+	github.com/onflow/flow-core-contracts/lib/go/templates v0.15.0
```

Comment:
How come this is a "downgrade" from a commit above `v1.2.4` to `v0.15.0`?

Reply:
`v1.2.3` was a tag that was pushed out of semver order a few months ago, which confuses the Go tooling. See https://github.com/onflow/flow-core-contracts/pull/398, which is retracting the incorrectly tagged version, for more context.